package.json

```diff
@@ -50,17 +50,16 @@
     "lint": "eslint",
     "test": "scripts/test.sh",
     "package": "npm ci --ignore-scripts && prebuild",
-    "install": "prebuild-install || cmake-js rebuild",
+    "install": "prebuild-install || cmake-js compile",
     "rebuild": "npm run rebuild-changes",
     "build": "npm run build-changes",
     "build-changes": "cmake-js build --debug",
-    "rebuild-changes": "cmake-js rebuild --debug && cd tests && npm install",
+    "rebuild-changes": "cmake-js rebuild --debug",
     "eslint": "npm install --ignore-scripts && npm run lint .",
     "license-check": "npm install && license-checker --exclude \"MIT,ISC,BSD,Apache-2.0,BSD-2-Clause,BSD-3-Clause,WTFPL,Unlicense,(MIT AND CC-BY-3.0)\" | node scripts/handle-license-check.js",
     "jsdoc:clean": "rimraf ./docs/output",
     "jsdoc": "npm install --ignore-scripts && npm run jsdoc:clean && jsdoc -u docs/tutorials -p package.json -c docs/conf.json",
-    "prenode-tests": "npm install --build-from-source && cd tests && npm install",
```

Comment:
So now you'd have to `(cd test && npm i)`, before calling `npm run node-tests`..
Does our CI know that?

Reply:
Seem like the tests are passing, they didn't use this script.

Reply:
Valid point 🤦‍♂️